### ISSUE TITLE
With --experimental, traceur generates wrong commonjs output since version 0.8.2

### ISSUE THREAD

**Issue by lgersman** [google/traceur-compiler#1757]
hi,

since version 0.8.2 (up to current version 0.8.4) traceur prepends

"use strict";
$traceurRuntime.options.symbols = true;

in **each** output file when using the following commandline option : 

````
traceur --experimental \
      --array-comprehension=true \
      --type-assertions \
      --type-assertion-module ../lib/assert/assert \
      --modules commonjs \
      --debug \
      --dir target src 
````

this seems not correct, isnt it ? 

now when i am trying to start such a transpiled file using 

````
node file.js
````
the js file has'nt any chance to load the traceur runtime  because traceur prepends the reference to 

````
$traceurRuntime.options.symbols = true;
````

right before the require statement to load the traceur runtime.

anyway - setting this traceur options again and again in every file cannot be the right way ...

up to version 0.0.81 everything was fine.

**Comment by arv** [comment]
Work around. Don't use experimental.

**Comment by arv** [comment]
Fixed by 67180b377559adeb330260e95b878afb99e826af
